netmap/types.proto

```diff
@@ -208,6 +208,18 @@ message NodeInfo {
   //   Must contain a comma-separated list of multi-addresses.
   // * Version
   //   Node implementation's version in a free string form.
+  // * VerifiedNodesDomain
+  //   Confirmation of admission to a group of storage nodes.
+  //   The value is the domain name registered in the NeoFS NNS. If attribute
+  //   is specified, the storage node requesting entry into the NeoFS network
+  //   map with this attribute must be included in the access list located on
+  //   the specified domain. The access list is represented by a set of TXT
+  //   records: Neo script hashes from public keys. To be admitted to the
```

Comment:
> Neo script hashes from public keys.

is it an understandable description from the Neo point of view, @roman-khimov?

Reply:
More or less. "address" might be a bit more appropriate since that's what we are going to store.

Reply:
changed to "addresses"